pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.6.1,<3.7"
-dcicutils = "^3.1.0"
+dcicutils = "^3.6.1"
```

Comment:
Is this the right version? I guess you'll get a later version because of the caret, but I meant do you need a beta of dcicutils (for python 3.7/3.8 version of dcicutils, for example)? Maybe there's nothing in there you need.